syntax = "proto3";

package example.simple;

option go_package = "./src/simple";
option java_package = "com.example.options";
option java_multiple_files = true;

message SimpleMessage {
    int32 id = 1;
    bool is_simple = 2;
    string name = 3;
    repeated int32 sample_list = 4;
}
